.changeset/fuzzy-bees-doubt.md

```diff
@@ -0,0 +1,51 @@
+---
+'@clerk/elements': minor
+---
+
+Support passkeys in SignIn flows.
+APIs introduced:
+- `<SignIn.Action passkey />`
```

Comment:
I'm more keen on `<SignIn.Passkey>`, but I'm open to what @BRKalow and @LekoArts think.

Reply:
Gut feeling-wise I'd probably also go with `<SignIn.Passkey>`

Reply:
The action should ideally be a verb. If it's more similar to `SignIn.Connection` in its function, then I agree it should be `<SignIn.Passkey>`